src/model/enhance-numbered-lists.ts

```diff
@@ -37,7 +37,13 @@ const extractH3 = (element: CAPIElement): string => {
 	const textElement = element as TextBlockElement;
 	const frag = JSDOM.fragment(textElement.html);
 	if (isFalseH3(element)) {
-		return frag.firstElementChild?.textContent || '';
+		return (
+			frag.firstElementChild?.innerHTML
+				.split('<strong>')
+				.join('')
+				.split('</strong>')
+				.join('') || ''
```

Comment:
not ideal, but its the simplest way of removing all strong tags from the text while still keeping other HTML tags and text in between strong tags